blackbox/docs/src/doc_tests/process_test.py

```diff
@@ -380,10 +380,5 @@ def exec_select_queries(client, is_active, errors, finished):
             try:
                 client.sql('select name, count(*) from t1 group by name')
             except Exception as e:
-                # For now we make sure that queries are not killed, but a
-                # TableUnknownException can happen if the queries still target
-                # the node that is being decommissioned and the cluster state
-                # update is too slow (and the internal 3 retries all happen before)
-                if 'TableUnknownException' not in str(e):
```

Comment:
I'm not sure if this is now really fixed. I ran it for quite a while but as it is with race conditions one can never be sure.